Sources/Player/Common/Extensions/AVPlayer.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+import MediaPlayer
+
+extension AVPlayer {
+	func seek(to position: Double) async -> Bool {
```

Comment:
[question] why have we moved this one out of the its file?

Reply:
Same answer to both!
Both extensions were part of files that I'm duplicating, so I moved them out to their own file so I could use them from both implementations without errors from duplicated symbols.